pkg/oauth/github/github.go

```diff
@@ -120,16 +120,23 @@ func (c *OAuthClient) GetUser(ctx context.Context) (*model.User, error) {
 	return &model.User{
 		Username:  user.GetLogin(),
 		AvatarUrl: user.GetAvatarURL(),
-		Role: &model.Role{
-			ProjectId:   c.project.Id,
-			ProjectRole: role,
-		},
+		Role:      role,
 	}, nil
 }
 
-func (c *OAuthClient) decideRole(user string, teams []*github.Team) (role model.Role_ProjectRole, err error) {
+func (c *OAuthClient) decideRole(user string, teams []*github.Team) (role *model.Role, err error) {
 	var found bool
 
+	role = &model.Role{
+		ProjectId:        c.project.Id,
+		ProjectRbacRoles: make([]string, 0, len(teams)),
+	}
+	groups := c.project.GetAllUserGroups()
+	roles := make(map[string]string, len(groups))
+	for _, g := range groups {
+		roles[g.SsoGroup] = g.Role
```

Comment:
Is this ok?
Is it possible if multiple groups with the same SsoGroup but have different Roles?

Reply:
Nope, It's not possible to use the existing name when somebody creates the Ssogroup.

Reply:
But this is including old and new groups then I think it could be possible for the old role to override the new role.
wdyt?
https://github.com/pipe-cd/pipecd/blob/master/pkg/model/project.go#L387-L405

Reply:
Good point, but we restricted it when adding a new user group.
Hence that value would be unique including old RBAC, I guess.
https://github.com/pipe-cd/pipecd/blob/2a551844a505c7a7f859a932c558f4660017281f/pkg/model/project.go#L372